src/main/kotlin/strdcoders/mss/services/RuleFlowRunner.kt

```diff
@@ -0,0 +1,10 @@
+package strdcoders.mss.services
+
+import org.springframework.stereotype.Service
+import strdcoders.mss.dto.PartitionMonitorDto
+
+@Service
+class RuleFlowRunner {
```

Comment:
Should have interface

Reply:
https://github.com/STRDCoders/mss/pull/2/commits/76c5871501eb636211210d1b0cf5ef31a2e41c47